config/migrations/2024/20241113095422-clean-up-verkiezingen/20241113110216-purge-kieslijsten.sparql

```diff
@@ -0,0 +1,19 @@
+PREFIX mandaat: <http://data.vlaanderen.be/ns/mandaat#>
+
+DELETE {
+  GRAPH ?g {
+    ?lijst ?lp ?lo .
+    ?res mandaat:isResultaatVoor ?lijst .
```

Comment:
What is `res`? Think you can do not have to worry about the length of your variable => verkiezingsResultaat or verkiezingsRes